### DISCUSSION TITLE
1499 Playbook Rules

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#8244] [Unsupported Versions] [answered]
Version 2.3.130 
Distributed GRID
Air gapped 

I recently stood up a distributed grid using SO2.3.130 iso in an air-gapped environment. After looking over the Playbook rules I see about 1499 of them. After confirming that no one has imported them manually I can't seem to find the source. Is this a new thing or an error on my part?

![image](https://user-images.githubusercontent.com/70167373/177564863-45b965d4-ff1a-428a-8465-3b932e8f8442.png)

This is what the /opt/so/rules/elastalert/playbook directory looks like before enabling rule 1326

![image](https://user-images.githubusercontent.com/70167373/177577391-5326df11-f8b0-47f6-8a1f-89d3efb50911.png)

Rule 1326

![image](https://user-images.githubusercontent.com/70167373/177577580-884c23c9-69da-4c8e-9bd6-9a30c4b17149.png)

After enabling it.

![image](https://user-images.githubusercontent.com/70167373/177577814-f21a25d0-5e77-4af1-bf84-fbb5a7fdfceb.png)

As stated before I have no idea how these got into our playbook. Still tracking that down. Any thoughts? I pulled this ISO on June 22, 2022 13:40 in the afternoon.

**Comment by isaacgolding** [comment, marked as answer]
There is a collection of draft plays that are installed with the system.  It is referenced in the docs.   https://docs.securityonion.net/en/2.3/playbook.html

**Reply by ColeVan** [reply]
I thought that SO2 shipped with about 500 plays not 1499??

**Reply by isaacgolding** [reply]
I just reinstalled a lab standalone box 2 hours ago and there are 1544 draft plays that I activated.

**Reply by ColeVan** [reply]
So this must come with the new 2.3.130 iso then. When I updated from 2.3.110 to 2.3.130 in a non-air gapped instance I did not get the additional rules.

**Reply by isaacgolding** [reply]
I can't speak to when the rules were onboarded.  I just know they are there every time I do an install.   And none of my installs are air-gapped.

**Reply by ColeVan** [reply]
Just ran so-playbook-update and now I have the new rules in my standalone instance as well.

![image](https://user-images.githubusercontent.com/70167373/177630692-1b4f4e17-c7ad-4cb8-a7b8-1c3cecaba65f.png)

**Reply by defensivedepth** [reply]
@ColeVan There was a change with .130 that points so-playbook-update directly to the upstream Sigma repo, so you should be getting new/updated rules more regularly now.
